addOns/wappalyzer/src/main/javahelp/org/zaproxy/zap/extension/wappalyzer/resources/help/toc.xml

```diff
@@ -6,7 +6,9 @@
 <toc version="2.0">
 	<tocitem text="ZAP User Guide" tocid="toplevelitem">
 		<tocitem text="Add Ons" tocid="addons">
-			<tocitem text="Technology Detection" image="addon.wappalyzer.icon" target="addon.wappalyzer.wappalyzer" />
+			<tocitem text="Wappalyzer - Technology Detection" image="addon.wappalyzer.icon" target="addon.wappalyzer">
```

Comment:
This will change the link of the website.

Reply:
I was referring to `Technology Detection` → `Wappalyzer - Technology Detection`, the map ID is not relevant.

Reply:
Oh. Okay I can unchange that part.

Reply:
I don't mind the change just saying so other places are updated accordingly (e.g link in add-on manifest).

Reply:
Naw I don't think it's necessary.